src/App.tsx

```diff
@@ -1,7 +1,30 @@
+import 'App.css';
 import { RouterProvider } from 'react-router-dom';
+import { useEffect } from 'react';
 import { router } from 'router';
-import 'App.css';
 
-const App = () => <RouterProvider router={router} />;
+const App = () => {
+  useEffect(() => {
+    const audio = document.querySelector('audio');
+    document.addEventListener(
+      'click',
```

Comment:
I suggest lowering initial audio value. My ears has been damaged thanks to volume level 🧏🏻‍♂️🦻🏻
```suggestion
    const audio = document.querySelector('audio');
    audio.volume = 0.7
    document.addEventListener(
      'click',
```